roles/install_java/molecule/resources/inventory/group_vars/all.yml

```diff
@@ -0,0 +1 @@
+---
```

Comment:
this empty file is required otherwise molecule complains that the inventory doesn't exist